exalearn-api/ExaLearn.WebApi/appsettings.json

```diff
@@ -8,7 +8,7 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "DbContext": "Server=localhost;Port=5432;Username=postgres;Password=Pass12345;Database=exalern_db",
+    "DbContext": "Server=localhost;Password=123456;Port=5432;Username=postgres;Password=123;Database=exalern_db",
```

Comment:
please remember - NEVER more commit the sensitive information as an user name and a password and NEVER more push it to remote repository. It is strictly prohibited for a real project and can cause your dismiss and penalty. You can leave the sensitive data in your local repository